### PR TITLE
Modified docs/References/SubscriptionMgt.md

### PR DESCRIPTION
**Pull request by isa-sen**
added the description of GetQuantity() method to the SubscriptionMgt page

### CODE REVIEW

File: docs/References/SubscriptionMgt.md [hunk 1]
```diff
@@ -267,6 +267,65 @@ GetPriceName(SecretKey:Text;ProductID:Text[100]):Text
 ```sql
 TBD
 ```
+### GetQuantity (Method)
+Returns the quantity of product.
+#### GetQuantity(Text,Text[100])
+![](https://img.shields.io/badge/version-v1.0.0.0-blue)
```
[the remark this answers is not in the document]
Comment (isa-sen) [reply docs/References/SubscriptionMgt.md]: Hi @vody , this is done.

File: docs/References/SubscriptionMgt.md [hunk 2]
```diff
@@ -267,6 +267,65 @@ GetPriceName(SecretKey:Text;ProductID:Text[100]):Text
...
+```sql
+[NonDebuggable]
+GetQuantity(SecretKey:Text;ProductID:Text[100]):Integer
```
[the remark this answers is not in the document]
Comment (isa-sen) [reply docs/References/SubscriptionMgt.md]: Hi @vody , I modified it's return value to BigInteger and completed changes.

File: docs/References/SubscriptionMgt.md [hunk 3]
```diff
@@ -267,6 +267,65 @@ GetPriceName(SecretKey:Text;ProductID:Text[100]):Text
 ```sql
 TBD
 ```
+### GetQuantity (Method)
+Returns the quantity of product.
```
[the remark this answers is not in the document]
Comment (isa-sen) [reply docs/References/SubscriptionMgt.md]: Hi @vody , this is done.

**Comment by vody** [comment]
@isa-sen, please resolve a merge conflict and will be happy to push it.

**Comment by isa-sen** [comment]
@vody, I resolved the conflicts, can you please check it again?

**Pull request merged**